README.md

```diff
@@ -1,7 +1,11 @@
-# microprofile-starter
+# MicroProfile Starter
 
 [![Gitter](https://badges.gitter.im/eclipse/microprofile-starter.svg)](https://gitter.im/eclipse/microprofile-starter?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
+## IDE Extensions and tools
+
+Ecosystem of tools and IDE extensions is hosted at [MicroProfile Starter Extensions](https://github.com/microprofile-starter-extensions) GitHub organization.
```

Comment:
Plase link to https://github.com/MicroShed/mp-starter-vscode-ext and https://github.com/MicroShed/mp-starter-intellij-ext

Reply:
@Karm Can you redo the PR with latest URLs of the IDE extensions?